src/main/perl/FileEditor.pm

```diff
@@ -47,7 +47,15 @@ the class constructor.
 =item new
 
 Returns a new object it accepts the same arguments as the constructor
-for C<CAF::FileWriter>
+for C<CAF::FileWriter> with one additional option: C<source>. This
+option, when present, must be a file name whose contents will be used
+as the initial contents for the edited file if the source modification time
+is more recent than the edited file modification time. This allows to rebuild
+the file contents based on a new version of the reference/template file. 
```

Comment:
remove the `/template`, nothing is templated here. i would also not use the term `reference` but stick with `source` to avoid confusion